src/components/Clock.tsx

```diff
@@ -0,0 +1,60 @@
+import { Component } from 'react';
+
+interface Props {
+  name: string;
+}
+
+interface State {
+  time: Date;
+}
+
+export class Clock extends Component<Props, State> {
+  // private interval: ;
+  state = {
+    time: new Date(),
+  };
+
+  intervalId = 0;
+
+  componentDidMount() {
+    this.intervalId = window.setInterval(() => {
+      // eslint-disable-next-line no-console
+      console.info(new Date().toUTCString().slice(-12, -4));
+      this.setState({
+        time: new Date(),
+      });
+    }, 1000);
```

Comment:
I think it will be better to create a variable for new Date() then creating to separate objects